languages.js

```diff
@@ -0,0 +1,44 @@
+exports.english = {
```

Comment:
I thought to change it to default it's a good idea because we may have several English packs.